src/server/upgrade/index.md

```diff
@@ -10,6 +10,39 @@ Make sure to always back up your database data before doing a migration.
 
 [[toc]]
 
+## Migration guide from 2024.4.x to 2025.2.x
```

Comment:
Can you please add also section about generating new security hashes? 
https://github.com/MerginMaps/docs/pull/542

These are mandatory. Also existing user sessions will be invalidated after upgrade (login required).